### ISSUE TITLE
💬 [Support Request] Gamemasters don't have power to cliffwalk

### ISSUE THREAD

**Issue by ClassicW** [cmangos/issues#3006]
### Core Version

mangos-classic

### Support Request Details

The Gamemasters can no longer climb the mountains.

![image](https://user-images.githubusercontent.com/71378304/173887727-2fe7d81e-b4c9-441f-9d33-8c84f9b5d08b.png)

this is due to this commit

https://github.com/cmangos/mangos-classic/commit/5fce1eb98a1b4cbd0b19035541604cc695e437bf

### Steps to Reproduce

1.go to cliff with gm
2.
3.
4.

### Core SHA1 Commit Hash

c07a5962627101606da7b323043f6a5d1c411bcf

### Database SHA1 Commit Hash

f4afa016d1c05397f47cc8d3fdaba39aa993cca7

### Server Operating System

linux

### Client Version

1.1.12 (Classic)

### Client Operating System

linux

**Comment by celguar** [comment]
This functionality was removed from gm on command recently. I guess we need to add a separate wallclimb command to make it work again.

**Comment by insunaa** [comment]
This is intended behaviour, closing.

**Comment by AnonXS** [comment]
I've to say, as a feature request i'd also like having this back, especially while deving in classicmangos without fly command this is super useful.

**Comment by insunaa** [comment]
Fair enough. We were abusing a system that was really not meant to be abused to enable that functionality, but I agree that it was useful.
Might make sense to add it to a separate command

**Comment by ClassicW** [comment]
Without the fly control, it becomes very fast a handicap
